packages/utils/src/node/fetchPublicKeys/fetch.public.keys.js

```diff
@@ -29,24 +30,22 @@ const getPublicKeys = async (connectUserPoolId) => {
   }
 }
 
-const syncKeys = async () => {
-  if (!fs.existsSync('./config.json')) {
+module.exports = async () => {
```

Comment:
Again, no big deal but file and folder names are kebab-case across the repo by convention.